lineapy/plugins/kubeflow_pipeline_writer.py

```diff
@@ -175,7 +175,7 @@ def get_rendered_task_definitions(
                 loading_blocks=loading_blocks,
                 pre_call_block="",
                 call_block=task_def.call_block,
-                post_call_block="",
+                post_call_block=task_def.post_call_block,
```

Comment:
same thing for pre_call_block makes sense?

Reply:
also not sure how jinja tackles [None]. Does it render as the string "None" or skips it? if its the former, we should use `task_def.post_call_block or ""` here.